recipes-noop/halservices/audio-service-rpi.bb

```diff
@@ -0,0 +1,16 @@
+SUMMARY = "No Operation Recipe for audioserver"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${MANIFEST_PATH_PLATFORM}/LICENSE;md5=5bfad6e034e497ee148eec56e175c6e8"
+
+PROVIDES = "virtual/audio-service virtual/vendor-audio-service"
+RPROVIDES:${PN} = "virtual/audio-service virtual/vendor-audio-service"
+
+ALLOW_EMPTY:${PN} = "1"
+
+PACKAGE_ARCH = "${VENDOR_LAYER_EXTENSION}"
```

Comment:
Can we move PACKAGE_ARCH to https://github.com/rdkcentral/meta-rdk-raspberrypi/blob/develop/conf/include/vendor_pkg_versions.inc so that recipe will remain "Bitbake" specific.